codecov.yml

```diff
@@ -0,0 +1,16 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: default
+  require_changes: true  # if true: only post the comment if coverage changes
+  require_base: no        # [yes :: must have a base report to post]
+  require_head: yes       # [yes :: must have a head report to post]

```

Comment:
If I understand this correctly, it will show a comment if:
- The branch being merged in has changed the coverage in some way
- The branch being merged in has a codecov report available
- Whether or not the merge-target (usually `master`) has a codecov report

I think `require_changes` should be `false` - I like the idea of having a little indication of "Coverage was not affected by this PR. I don't have strong feelings though :) happy to leave it as-is for now and update later if we feel the need